### PR TITLE
Update documentation for Attribute object

### PR DESCRIPTION
**Pull request by vdancik**

### CODE REVIEW

**Review by edeutsch — CHANGES_REQUESTED** [review]
Seems pretty good, but please fix the too-long-lines. Max line length is 79 chars. Also on line 370 'referably' -> 'preferably'

**Review by CaseyTa — APPROVED** [review]

**Review by edeutsch — APPROVED** [review]
looks good, thanks!

File: TranslatorReasonerAPI.yaml [hunk 1]
```diff
@@ -336,35 +336,41 @@ components:
...
+          $ref: '#/components/schemas/CURIE'
           description: >-
-            CURIE of the semantic type of the attribute, from the EDAM ontology
-            if possible. If a suitable identifier does not exist, enter a
-            descriptive phrase here and submit the new type for consideration
-            by the appropriate authority.
+            CURIE of the semantic type of the attribute. For properties
+            defined by the Biolink model this should be a biolik CURIE,
+            otherwise, if possible, from the EDAM ontology. If a suitable
+            identifier does not exist, enter avdescriptive phrase here and
```
Comment (CaseyTa) [inline TranslatorReasonerAPI.yaml:361]: Minor typo with "avdescriptive"

**Review by CaseyTa — CHANGES_REQUESTED** [review]

File: TranslatorReasonerAPI.yaml (hunk 1 above)
Comment (vdancik) [reply TranslatorReasonerAPI.yaml]: typo is now fixed

**Review by CaseyTa — APPROVED** [review]

**Review by bill-baumgartner — APPROVED** [review]

**Review by patrickkwang — APPROVED** [review]

**Pull request merged**